plugins/genocide/genocide.cpp

```diff
@@ -12,25 +12,37 @@ class genocide : public bz_Plugin
     {
         return "Genocide";
     }
-    virtual void Init ( const char* c );
+    virtual void Init ( const char* config );
     virtual void Cleanup ( void );
     virtual void Event ( bz_EventData *eventData );
+
+    bool disableSuicide = false;
```

Comment:
Within the _RogueGenocide_ plugin, it was done as such.
Any recommendations? Since I've seen both **private** and **protected** used in a few other plugins
For instance, _playHistoryTracker_ uses **protected**, _regFlag_ uses **private** and _airspawn_ uses **public.**